plugins/+ciplugins/+github/WorkflowCommandsPlugin.m

```diff
@@ -0,0 +1,22 @@
+classdef WorkflowCommandsPlugin < matlab.buildtool.plugins.BuildRunnerPlugin
+
+%   Copyright 2024 The MathWorks, Inc.
+
+    methods (Access=protected)
+
+        function runTask(plugin, pluginData)
+           % Add Github workflow command for grouping the tasks
+           disp("::group::" + pluginData.TaskResults.Name + " - task");
+
+           runTask@matlab.buildtool.plugins.BuildRunnerPlugin(plugin, pluginData);
+
+           % Add Github workflow command ::error:: if the task is failed
+           if pluginData.TaskResults.Failed
+              disp("::error::" + pluginData.TaskResults.Name + " - task");
```

Comment:
Should we consider a different message here? It doesn't give much information and looks a little informal next to the other annotations
<img width="910" alt="Screenshot 2024-05-16 at 10 55 35 AM" src="https://github.com/matlab-actions/run-build/assets/103441853/d13bf024-0810-40f7-8208-f72011cd8370">

Reply:
Do you mean it should be something like `::error:: <taskname> - task execution failed`

Reply:
My preference is something like `::error:: Error: <taskname> task failed`

Reply:
I personally feel the annotation in the summary is really helpful with this. iits good value add with simple change. i can change the message for now to make it bit clearer. does that sounds good

Reply:
changed it to ::error::<taskname> task failed